creator-node/src/utils.js

```diff
@@ -77,9 +82,13 @@ const wait = (ms) => new Promise((resolve, reject) => setTimeout(() => reject(ne
 /** Cat single byte of file at given filepath. */
 const ipfsSingleByteCat = (path, req) => new Promise(async (resolve, reject) => {
   const start = Date.now()
-  let ipfs = req.app.get('ipfsAPI')
+  let ipfs = req.app.get('ipfsLatestAPI')
```

Comment:
should we have a fallback to the old api or something? otherwise why include both?

Reply:
nvm, figured this out